### PR TITLE
[Docs] PR for Update Async Replication Lag Section

### PR DESCRIPTION
**Pull request by lizayugabyte**

### CODE REVIEW

File: docs/content/preview/deploy/multi-dc/async-replication.md [hunk 1]
```diff
@@ -132,15 +132,17 @@ Replication lag is computed at the tablet level as follows:
 
 *hybrid_clock_time* is the hybrid clock timestamp on the source's tablet-server, and *last_read_hybrid_time* is the hybrid clock timestamp of the latest record pulled from the source.
 
-An example script [`determine_replication_lag.sh`](/files/determine_replication_lag.sh) calculates the replication lag. The script requires the [`jq`](https://stedolan.github.io/jq/) package.
+An example script `determine_replication_lag.sh` located in the `files` directory calculates the replication lag. The script requires the jq package.
```
Comment (mrajcevic01) [inline docs/content/preview/deploy/multi-dc/async-replication.md:135]: From [the slack thread](https://yugabyte.slack.com/archives/CHC5RSY74/p1650383646819669?thread_ts=1650380816.965229&cid=CHC5RSY74) it seems that this script does not work so we should remove any mention of it completely

File: docs/content/preview/deploy/multi-dc/async-replication.md [hunk 2]
```diff
@@ -132,15 +132,17 @@ Replication lag is computed at the tablet level as follows:
...
+The following example generates a replication lag summary for all tables on a cluster:
 
 ```sh
 ./determine_repl_latency.sh -m 10.150.255.114,10.150.255.115,10.150.255.113
 ```
 
-To obtain a summary of all command options, execute `determine_repl_latency.sh -h` .
+You can also request an individual table.
+
+To obtain information about the overall maximum lag, you should check `/metrics` or `/prometheus-metrics` for `async_replication_sent_lag_micros` or `async_replication_committed_lag_micros` and take the maximum of these values across each source's T-Server.
```
Comment (mrajcevic01) [inline docs/content/preview/deploy/multi-dc/async-replication.md:145]: Do we want to point to docs on setting up node exporter/prometheus in the first place as well? This subsection is under YB core which means they will have to [set up prometheus manually](https://docs.yugabyte.com/preview/explore/observability/prometheus-integration/linux/).

File: docs/content/preview/deploy/multi-dc/async-replication.md (hunk 1 above)
Comment (lizayugabyte) [reply docs/content/preview/deploy/multi-dc/async-replication.md:135]: The ticket description says that `determine_repl_latency.sh` shoudl be removed. 

There's nothing about `determine_replication_lag.sh`

Comment (lizayugabyte) [reply docs/content/preview/deploy/multi-dc/async-replication.md:135]: Marko Rajcevic via slack:

I believe neither works
based off what I am seeing in the thread
Alex is under the impression its the same file but was just renamed
I just asked Tyler to confirm in the thread

Comment (lizayugabyte) [reply docs/content/preview/deploy/multi-dc/async-replication.md:135]: I removed any mentioning of these two scripts

**Comment by lizayugabyte** [comment]
@mrajcevic01 Could you take a look at the changes and approve the PR, please?

**Review by mrajcevic01 — APPROVED** [review]

**Pull request merged**